Mtneer/mushu
# Edit transaction category

# Description
As a user I would like to be able to modify the category of a Transaction.

**Given** the user is viewing the Transaction list
**When** they select the option to edit a Transaction
**Then** the user should be directed to a form and given the ability to change the Transaction's category

**Given** the user is finished updating the Transaction information
**When** they click the Save button
**Then** the updated Transaction should be saved to the database
**And** the user should be redirected to the Transaction List

**Given** the user has decided not to edit the Transaction
**When** they click the Cancel button
**Then** the user should be redirected back to the list page

## Type of change
Please delete options that are not relevant.

# Testing Instructions
- pull down new branch
- start web API by launching Visual Studio and clicking play
- start React App by running npm start in the client directory
- login to the app using "admin@admin.com" and "password1"
- select the "Transactions" tab
- click the "Edit" button on the right side of one of the transaction entries. Make note of which entry.
- change the category using the dropdown form.
- click save
- verify that you are navigated back to the transactions list and that the transaction you edited displays the changed category.

# Checklist:
